patches/@rainbow-me+swaps+0.1.15.patch

```diff
@@ -0,0 +1,25 @@
+diff --git a/node_modules/@rainbow-me/swaps/.DS_Store b/node_modules/@rainbow-me/swaps/.DS_Store
```

Comment:
why do we patch our own repo?

Reply:
we need to point swaps sdk to a staging endpoint for now until we release it, i'll add it to the description